galasa-ui/src/utils/interfaces/FrontEndClient.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+export interface FrontEndClient {
+    clientName: string;
+    lastLogin: string; // Use `Date` type if you want to work directly with Date objects
```

Comment:
Comment copied from somewhere? :)

It's a good suggestion to work with the Date directly though (if you can!)